release.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  branches: ['master'],
+  plugins: [
+    ['@semantic-release/commit-analyzer', { preset: 'conventionalcommits' }],
+    '@semantic-release/npm',
```

Comment:
Over in Chroma, we specify `pkgRoot`, but it requires a `package.json` file according to the docs.

> Note: The pkgRoot directory must contain a package.json. The version will be updated only in the package.json and npm-shrinkwrap.json within the pkgRoot directory.

When installing Chromicons, the package has the following contents.  Due to this, I believe we can leave this as is, but please correct me if I'm wrong here 🙇 
<img width="233" alt="Screen Shot 2021-07-28 at 1 15 11 PM" src="https://user-images.githubusercontent.com/8069555/127367476-494abc03-cdac-4c7f-a7ad-c0972fa8d41d.png">

Reply:
If your goal is to have the published package look like the file structure you posted above, this looks great!